src/server/game/World/World.cpp

```diff
@@ -194,6 +199,59 @@ const char* World::GetMotd() const
     return m_motd.c_str();
```

Comment:
To resolve the conflict in this file, remove your line numbers 190-201 ( `motd` changes in TC).